.github/workflows/build_and_test.yml

```diff
@@ -214,7 +214,6 @@ jobs:
     uses: ./.github/workflows/build_reusable.yml
     with:
       afterBuild: |
-        pnpm install
```

Comment:
This might be needed still it looks like we fail to create binary links during initial install for some reason 🤔  

https://github.com/vercel/next.js/actions/runs/14761060792/job/41441744133?pr=78720#step:27:253

Reply:
Apparently the binaries don't exist until after `pnpm build` is run at least once...

This is kinda bad, but not awful. It should at least be documented (which my updated PR does).